cosmos/ingestion/ingest/utils/table_extraction.py

```diff
@@ -230,22 +263,33 @@ def _extract_tables(self) -> List[Tuple[str, pd.DataFrame]]:
             yield (x.pkl_path, x.extract_table()[0])
 
     def extract_csvs(self) -> None:
-        """save each table location df as a csv with that pdfs name and an incrementing int"""
+        """save each table location df as a csv with that pdfs name and an incrementing int,
+        the table extracted by pdfplumber ends with the '_pdfplumber.csv' extension """
         for pkl_path, df in self._extract_tables():
             csv_path = pkl_path[:-4] + '.csv'
+            csv_path_pdfplumber = pkl_path[:-4] + '_pdfplumber.csv'
             try:
-                df.to_csv(csv_path, index=False)
+                df[0].to_csv(csv_path, index=False)
             except AttributeError:
                 logging.info(f'no df to csv: {pkl_path}')
+            try:
+                df[1].to_csv(csv_path_pdfplumber, index=False)
+            except AttributeError:
+                logging.info(f'no df to csv: {csv_path_pdfplumber}')
 
     def extract_pickles(self) -> None:
         """save each table location df as a pickled df with that pdf's name and an incrementing int
-        AND update the _tables.parquet with the path to those pkl files per row"""
+        AND update the _tables.parquet with the path to those pkl files per row,
+        the table extracted by pdfplumber ends with the '_pdfplumber.pkl' extension"""
         for pkl_path, df in self._extract_tables():
             try:
-                df.to_pickle(pkl_path)
+                df[0].to_pickle(pkl_path)
             except AttributeError:
                 logging.info(f'no table df to pickle: {pkl_path}')
+            try:
+                df[1].to_pickle(pkl_path[:-4] + '_pdfplumber.pkl')
+            except AttributeError:
+                logging.info(f'no table df to pickle: {pkl_path[:-4] + '_pdfplumber.pkl'}')   
```

Comment:
I used the script above and tested with `documents_5Feb_tables.parquet` and the COSMOS to Pdfplumber coordinate conversion does work. This is because Pdfplumber shares the same document dimensions with Camelot (779x595), so the existing [scale factor calculation](https://github.com/UW-COSMOS/Cosmos/blob/master/cosmos/ingestion/ingest/utils/table_extraction.py#L53) should work the same for Pdfplumber as for Camelot. 

However, I noticed two issues: 1. Pdfplumber with the default settings sometimes has trouble detecting tables with no graphical boundaries, even though the table region is correct. E.g., a table in 5ef119afa58f1dfd5209bd33.pdf 
![image](https://user-images.githubusercontent.com/47307188/230496729-ac85a678-fc57-4e05-9008-318aaf9b2194.png). The solution to this is using the `text` extraction strategy `page.extract_table(table_settings={'vertical_strategy':'text'})`, which deduces imaginary lines based on text locations regardless of whether there are graphical lines, but the extraction results could be wrong like in this csv file
[5ef119afa58f1dfd5209bd33.pdf.csv](https://github.com/UW-COSMOS/Cosmos/files/11174578/5ef119afa58f1dfd5209bd33.pdf.csv).

2. COSMOS may recognize other areas as tables and send inaccurate coordinates. For example, during testing, the region below was given in `documents_5Feb_tables.parquet` impling it is a table region, though it's a proof with matrix. While Camelot extracted the proof line by line, Pdfplumber could not extract this and threw an error
![image](https://user-images.githubusercontent.com/47307188/230499858-bf04aa3a-de06-4582-a6ee-c9803244c1c3.png). Another option could be using `page.find_tables(table_settings={'vertical_strategy':'text'})[0].bbox`, which is Pdfplumber's builtin function to return the coordinates of a table within a page, but this would mean we are ignoring the cordinates given by COSMOS.